presto-mysql/src/main/java/io/prestosql/plugin/mysql/MySqlClient.java

```diff
@@ -307,8 +307,9 @@ public void renameColumn(JdbcIdentity identity, JdbcTableHandle handle, JdbcColu
     @Override
     protected void copyTableSchema(Connection connection, String catalogName, String schemaName, String tableName, String newTableName, List<String> columnNames)
     {
+        // Create a temporary table not only include columns to be inserted but all columns,  Need them for `NOT NULL` check.
         String sql = format(
-                "CREATE TABLE %s LIKE %s",
+                "CREATE TABLE %s AS SELECT * FROM %s WHERE 0 = 1",
```

Comment:
If we change to CTAS from the current implementation, it breaks the existing behavior when GTID mode is on. We need to look for other approach (e.g. Use CREATE TABLE LIKE only when GTID mode is on or just adding cofiguration).

Reply:
do we have tests for this? or not possible? (sorry, dont remember)
(sadly, all checks paassd... )

Reply:
I see, `CTAS` not working in GTID mode. I found this issue  #2251

Reply:
> do we have tests for this? or not possible? (sorry, dont remember)
> (sadly, all checks paassd... )

@ebyhr just file an issue to add tests in GTID mode :) #4997